test/modules/standard/jama/TestMatrix.chpl

```diff
@@ -0,0 +1,403 @@
+use Jama;
+use Math;
+
+/** TestMatrix tests the functionality of the Jama Matrix class and associated decompositions.
+<P>
+Run the test from the command line using
+<BLOCKQUOTE><PRE><CODE>
```

Comment:
What do you think about a blended name like LinearAlgebra-Jama.chpl?  My slight hesitancy in calling it something completely generic is that if we change the "default" linear algebra interface down the road, we break code.  Whereas if, over time, we say "Yeah, this is *the* blessed linear algebra package for Chapel!" then we can rename this and just have people update a 'use' clause (or continue to support the alias for backwards compatibility).